docs/copilot/copilot-customization.md

```diff
@@ -131,14 +131,90 @@ To configure custom commit message generation instructions, use the `setting(git
 
 * Make it easy to share custom instructions with your team or across projects by storing your instructions in an external file. You can also version control the file to track changes over time.
 
+## Reusable prompt files (preview)
+
+Prompt files (_prompts_) let you build and share reusable prompt instructions with additional context. A prompt file is a Markdown file that mimics the existing format of writing prompts in Copilot Chat (for example, `Rewrite #file:x.ts`). This allows blending natural language instructions, additional context, and even linking to other prompt files as dependencies.
+
+While custom instructions help to add codebase-wide context to each AI workflow, prompt files let you add instructions to a specific chat interaction.
+
+Common use cases include:
+
+- **Code generation**: create reusable prompts for components, tests, or migrations (for example, React forms, or API mocks).
+- **Domain expertise**: share specialized knowledge through prompts, such as security practices, or compliance checks.
+- **Team collaboration**: document patterns and guidelines with references to specs and documentation.
+- **Onboarding**: create step-by-step guides for complex processes or project-specific patterns.
+
+### Prompt file examples
+
+- `react-form.prompt.md` - documents a reusable task for generating a form:
+
+  ```markdown
+  Your goal is to generate a new React form component.
+
+  Ask for the form name and fields if not provided.
+
+  Requirements for the form:
+  - Use form design system components: [design-system/Form.md](../docs/design-system/Form.md)
+  - Use `react-hook-form` for form state management:
+    - Always define TypeScript types for your form data
+    - Prefer *uncontrolled* components using register
+    - Use `defaultValues` to prevent unnecessary rerenders
+  - Use `yup` for validation:
+    - Create reusable validation schemas in separate files
+    - Use TypeScript types to ensure type safety
+    - Customize UX-friendly validation rules
+  ```
+
+- `security-api.prompt.md` - documents reusable security practices for REST APIs, which can be used to do security reviews of REST APIs:
+
+  ```markdown
+  Secure REST API review:
+  - Ensure all endpoints are protected by authentication and authorization
+  - Validate all user inputs and sanitize data
+  - Implement rate limiting and throttling
+  - Implement logging and monitoring for security events
+  …
+  ```
+
+### Usage
+
+To enable prompt files, configure the `setting(chat.promptFiles)` VS Code setting. Set it to `true` or use the `{ "/path/to/folder": boolean }` notation to provide specific folders where your prompt files are stored. Relative folder paths are resolved relative to the root folder(s) of your project, and the default value of `.github/prompts` is used if no other paths are specified.
```

Comment:
Updated the config value description here, feel free to update my updates 😄

Reply:
Reads great.